extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -58,40 +59,47 @@ export class AmpAudio extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    const viewer = Services.viewerForDoc(this.getAmpDoc());
+    viewer.whenFirstVisible().then(() => {
+      console.log('hello there');
+      const audio = this.element.ownerDocument.createElement('audio');
+      if (!audio.play) {
+        this.toggleFallback(true);
+        return Promise.resolve();
+      }
+
+      // Force controls otherwise there is no player UI.
+      audio.controls = true;
+
+      //TODO (nainar): Should we be overriding the user set values for
```

Comment:
@jridgewell should we force set `preload` and `autoplay` here and then propagate the user set attributes in `layoutCallback()` instead?